linchpin/cli/__init__.py

```diff
@@ -278,41 +281,27 @@ def _build(self, pf):
 
         """
 
-        ws = self.workspace
-
         provision_data = {}
 
         for target in pf.keys():
 
             provision_data[target] = {}
 
             if not isinstance(pf[target]['topology'], dict):
-                topology_data = (
-                    self.parser.process(self.find_topology(pf[target]["topology"]))) # noqa E501
+                topology_path = self.find_include(pf[target]["topology"])
+                topology_data = self.parser.process(topology_path, pf_data)
             else:
                 topology_data = pf[target]['topology']
 
             provision_data[target]['topology'] = topology_data
 
-
             layout_data = None
 
             if 'layout' in pf[target]:
                 if not isinstance(pf[target]['layout'], dict):
-                    self.set_evar('layout_file',
```

Comment:
set_evar('layout_file'..  is not longer needed?

Reply:
Nope. layou_file is now layout_data. Essentially, we could set the evar, but it's not used in the playbooks anymore.

Reply:
ok - +1 then once you fix the incorrect doc string in my previous comment